dll/win32/shell32/CDefViewBckgrndMenu.cpp

```diff
@@ -318,12 +281,7 @@ CDefViewBckgrndMenu::HandleMenuMsg2(UINT uMsg, WPARAM wParam, LPARAM lParam, LRE
 {
     if(m_folderCM)
```

Comment:
Could take the opportunity for adding space after "if" (and removing the curly brackets?)

Reply:
no.